src/main/java/Duke.java

```diff
@@ -1,10 +1,146 @@
+import java.util.*;
```

Comment:
general feedback would be to use more abstraction with a ui class and command class and by using javadocs for easier comprehension but overall good progress!

Reply:
thank you!